app/views/admin/_admin_options.html.erb

```diff
@@ -6,6 +6,29 @@
   <% elsif item.class == Bookmark %>
     <% item_type_name = ts("Bookmark") %>
   <% else %>
+    <li><%= link_to ts('Hide'), {:controller=>"admin/user_creations", :action=>"hide", :id=>item, :creation_type => item.class, :hidden=>'true'} %></li>
```

Comment:
I think something might've gone wrong while updating this branch -- the new Mark Not Spam/Mark Spam options should go down a few lines to be in the ul, not the part that defines the item type, and it looks like the old Hide and Edit Tags options snuck in up here as well.